setup.py

```diff
@@ -8,13 +8,15 @@
 
 def requirements_from_file(filename):
     """Parses a pip requirements file into a list."""
-    return [line.strip() for line in open(filename, 'r')
-            if line.strip() and not line.strip().startswith('--')]
+    with open(filename, 'r') as fid:
```

Comment:
Just a nitpick, but I'd call the variable `fd` for File Descriptor, not `fid`ŧ

Reply:
Sounds good, I've made this change. Thanks!